btree/btree.c

```diff
@@ -3164,7 +3194,7 @@ static struct m0_sm_state_descr btree_states[P_NR] = {
 
 static struct m0_sm_trans_descr btree_trans[256] = {
 	{ "create-init", P_INIT,  P_ACT  },
-	{ "create-act",  P_ACT,   P_DOWN },
+	{ "create-act",  P_ACT,   P_DONE },
         { "destroy", P_INIT, P_DONE},
```

Comment:
There seem to be spaces at the start. Use tab here to avoid conflicts.

Reply:
ok.